### ISSUE TITLE
Return Scroll - Banished Area keeps adding in shop

### ISSUE THREAD

**Issue by Rednor** [ronancpl/HeavenMS#433]
For some reason Return Scroll - Banished Area in the potion shop keeps on multiplying. I'm using the shopupdate.sql
![Maple0000](https://user-images.githubusercontent.com/48612569/54471097-76fc7580-47ff-11e9-8456-4485b11a7099.jpg)

**Comment by ronancpl** [comment]
Make sure to run the 3 SQL scripts in order (specified on readme) and just once each when you first create the server's SQL schema. They should appear only once on each shop it actually appears in.

Although, by running this SQL command `SELECT shopid, count(*) FROM shopitems WHERE itemid = 2030100 GROUP BY shopid;`, it appears there's currently a bug on the shop of NPC 9270022, having a duplicate of that item in there.

**Comment by Rednor** [comment]
I just reinstalled the server to fix the issue.
